### PR TITLE
Minor fix in man-page scap-as-rpm.8

### PR DESCRIPTION
**Pull request by havardAasen**
Update whatis entry and add some commas to fix grammar.
Replaced deprecated .HP macro with .TP.

### CODE REVIEW

File: utils/scap-as-rpm.8 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .TH scap-as-rpm "8" "November 2013" "scap-as-rpm" "System Administration Utilities"
 .SH NAME
-scap-as-rpm \- manual page for scap-as-rpm
+scap-as-rpm \- takes given SCAP input and create an RPM package
```
Comment (evgenyz) [inline utils/scap-as-rpm.8:3]:
```suggestion
scap-as-rpm \- takes given SCAP input(s) and packs them in an RPM package 
```

File: utils/scap-as-rpm.8 [hunk 2]
```diff
@@ -13,8 +13,9 @@ FILE [FILE ...]
 .PP
 Takes given SCAP input(s) and makes an RPM package that contains them. The
 result RPM can be installed using # yum install ./package\-name\-1\-1.rpm which
-will put the contents into \fI/usr/share/xml/scap\fP. No dependency on openscap or scapworkbench is enforced in the output package so you can use any SCAP\-capable
-scanner to evaluate the content.
+will put the contents into \fI/usr/share/xml/scap\fP. No dependency on openscap
+or scap\-workbench is enforced in the output package, so you can use any
```
Comment (evgenyz) [inline utils/scap-as-rpm.8:17]: I'd rather completely remove the mention of the SCAP Workbench.

File: utils/scap-as-rpm.8 [hunk 3]
```diff
@@ -13,8 +13,9 @@ FILE [FILE ...]
 .PP
 Takes given SCAP input(s) and makes an RPM package that contains them. The
 result RPM can be installed using # yum install ./package\-name\-1\-1.rpm which
-will put the contents into \fI/usr/share/xml/scap\fP. No dependency on openscap or scapworkbench is enforced in the output package so you can use any SCAP\-capable
-scanner to evaluate the content.
+will put the contents into \fI/usr/share/xml/scap\fP. No dependency on openscap
```
Comment (evgenyz) [inline utils/scap-as-rpm.8:16]:
```suggestion
will put the contents into \fI/usr/share/xml/scap\fP. No dependency on the OpenSCAP
```

File: utils/scap-as-rpm.8 (hunk 1 above)
Comment (havardAasen) [reply utils/scap-as-rpm.8]: Applied the suggestion, except for the end-of-line whitespace

File: utils/scap-as-rpm.8 (hunk 3 above)
Comment (havardAasen) [reply utils/scap-as-rpm.8]: Updated with correct capitalization on OpenSCAP, but couldn't get 'the' to fit in.

**Comment by evgenyz** [comment]
LGTM. Thank you!

**Review by evgenyz — APPROVED** [review]

**Pull request merged**